docs/api/api_blueprint.apib

```diff
@@ -3363,6 +3364,7 @@ Get a single event.
                     "privacy": "public",
                     "code-of-conduct": "example",
                     "state": "draft",
+                    "average-rating": null,
```

Comment:
You have to add in each and every event. I mean for eg. ```Get Event under an Order```, ```Get Event under a Stripe-Authorization``` etc. Add it everywhere you're expecting an event to be returned.

Reply:
Sure, doing it. 👍